help/help.adoc

```diff
@@ -11,24 +11,53 @@ This plugin was developed and tested against Windows Azure 2012.
 {CD} minimum version 4.0
 
 [[Windows_Certificates]]
-== Windows certificates
+== Working with Windows certificates
 
-Some procedures require a certificate to authenticate the request, you need to create one and upload it to Windows Azure:
+Azure procedures may require a Windows certificate to authenticate requests. To create and upload a certificate to Windows Azure:
 
-[[Create_a_X509_certificate]]
-== Create a x509 certificate
+. Generate an X.509 certificate using the Certificate Creation Tool (Makecert.exe) using the follow command:
++
+``makecert -sky exchange -r -n "CN=<CertificateName>" -pe -a sha1 -len 2048 -ss My "D:/<CertificateName>.cer"``
++
+NOTE: Makecert.exe is included with Visual Studio and Windows SDK installations. For more information, see link:http://msdn.microsoft.com/en-us/library/bfsktky3(VS.80).aspx[Microsoft's Makecert.exe page].
 
-Using Certificate Creation Tool (Makecert.exe): makecert -sky exchange -r -n "CN=<CertificateName>" -pe -a sha1 -len 2048 -ss My "D:/<CertificateName>.cer"
+. To upload the Windows certificate to Windows Azure, see link:https://azure.microsoft.com/en-us/documentation/articles/cloud-services-configure-ssl-certificate/[How to Add a Management Certificate to a Windows Azure Subscription].
 
-For more information: http://msdn.microsoft.com/en-us/library/bfsktky3(VS.80).aspx[Makecert].
+[[Plugin_Procedures]]
+== Plugin procedures
 
-[[Upload_the_certificate]]
-== Upload the certificate
+This plugin enables the following:
 
-https://azure.microsoft.com/en-us/documentation/articles/cloud-services-configure-ssl-certificate/[How to Add a Management Certificate to a Windows Azure Subscription].
+* Database management
+** <<Create-Or-Update-Database>>
+** <<Delete_Database>>
+** <<DeleteDatabase>>
 
-[[Plugin_Procedures]]
-== Plugin procedures
+* Group management
+** <<Create_or_Update_NetworkSecurityGroup>>
+** <<Delete_NetworkSecurityGroup>>
+** <<Create_or_Update_NetworkSecurityRule>>
+** <<Delete_NetworkSecurityRule>>
+
+* Network management
+** <<Create_or_Update_Subnet>>
+** <<Delete_Subnet>>
+** <<CreateVnet>>
+** <<DeleteVnet>>
+
+* VM management
+** <<CreateVM>>
+** <<Start_VM>>
+** <<Restart_VM>>
+** <<Stop_VM>>
+** <<Delete_VM>>
+
+* SQL and NoSQL operations
+** <<SQL_Operations>>
+** <<NoSQL_Operations>>
+
+* Resource management
+** <<Teardown>>
 
 IMPORTANT: For all parameter descriptions below, required parameters are shown in [.required]#bold italics#.
```

Comment:
We could remove this admonition, since we state "(Required)" for the required params below and none of them are bold/italic?